### PR TITLE
add couple of array value

### PR DESCRIPTION
**Pull request by Umangvaghela**
add couple of array value

### CODE REVIEW

File: src/Scaffold_Command.php [hunk 1]
```diff
@@ -902,16 +902,19 @@ private function pluralize( $word ) {
 			'/$/'                      => 's'
 		);
 
-		$uncountable = array( 'equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep' );
-
+		$uncountable = array('sheep', 'fish','deer','series','species','money','rice','information','equipment');
```
Comment (Sidsector9) [inline src/Scaffold_Command.php]: @Umangvaghela While you're working on this, can you please update this as per WordPress' coding standards?

1. Space after `,`
2. Space after `(` and before `)`

More details about coding standards can be found here - 
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/

File: src/Scaffold_Command.php [hunk 2]
```diff
@@ -902,16 +902,19 @@ private function pluralize( $word ) {
...
-		);
+			  'move'   => 'moves',
+			  'foot'   => 'feet',
+			  'goose'  => 'geese',
+			  'sex'    => 'sexes',
+			  'child'  => 'children',
+			  'man'    => 'men',
+			  'tooth'  => 'teeth',
+			  'person' => 'people',
+			  'valve'  => 'valves'
```
Comment (Sidsector9) [inline src/Scaffold_Command.php]: Last array item must end with a `,`

**Comment by Umangvaghela** [comment]
Hi @Sidsector9 

I have added a patch to remove white space issue to use PHP editor.

**Comment by miya0001** [comment]
Hi @Umangvaghela 

Thanks for PR!

Can you recover the sort order of elements?
Please add new element at the last if you want to add.

We have to check every element. 😓

File: src/Scaffold_Command.php [hunk 3]
```diff
@@ -902,15 +902,18 @@ private function pluralize( $word ) {
 			'/$/'                      => 's'
 		);
 
-		$uncountable = array( 'equipment', 'information', 'rice', 'money', 'species', 'series', 'fish', 'sheep' );
+		$uncountable = array( 'sheep', 'fish', 'deer', 'series', 'species', 'money', 'rice', 'information', 'equipment' );
```
Comment (miya0001) [inline src/Scaffold_Command.php]: Please recover sort order of the elements.

File: src/Scaffold_Command.php [hunk 4]
```diff
@@ -902,15 +902,18 @@ private function pluralize( $word ) {
...
 
 		$irregular = array(
-			'person' => 'people',
```
Comment (miya0001) [inline src/Scaffold_Command.php]: Please recover sort order of the elements.

File: src/Scaffold_Command.php (hunk 3 above)
Comment (Umangvaghela) [inline src/Scaffold_Command.php]: @miya0001 
$uncountable = array( 'sheep', 'fish', 'deer', 'series', 'species', 'money', 'rice', 'information', 'equipment' );
do you want this form 
$uncountable = array(   'deer','equipment', 'fish' , 'information',  'money', 'rice', 'series', 'sheep', 'species' );

Comment (Sidsector9) [inline src/Scaffold_Command.php]: @Umangvaghela what @miya0001 meant is that don't change the order of the default array. If you want to add new elements then add it to the end of the array.

**Comment by miya0001** [comment]
Anyway, is this pattern is finite?
Do you know any dictionary or so?

**Comment by miya0001** [comment]
It is same problem with #42, #44.
Please see discussion on the #44

**Comment by Umangvaghela** [comment]
@miya001 

Appreciate fresher contributor  effort.I know pluralization is totally wrong .I don't know who's check this.If we start to make this kind of complex structure that time we need more wisdom about this thing .I hope you working perfect without error.I know some member could not appreciate fresher contributor patch .I see same patch add member that time no issue are find and no need to explanation while new contributors try to  add patch that time member ask detail or example.It is okay .
Good by

**Comment by miya0001** [comment]
@Umangvaghela 

Oh, I apologize for lack of explanation in this topic.
I wrote a reason why I close this PR at #44 , so I forgot to say thank you in this topic.

Actually, we are really appreciate you, but I want to find the best way to solve the problem.
So, I already open a new issue which is find out by your effort.
https://github.com/wp-cli/scaffold-command/issues/48

Thanks!

**Pull request closed without merging**